app/src/userpages/utils/filters.js

```diff
@@ -0,0 +1,27 @@
+// @flow
```

Comment:
Can we simplify it? I wrote a little something, as a suggestion: https://github.com/streamr-dev/streamr-platform/commit/b00360745b5bed8946996654ade4ce7a74e5a435. Cheery-pick it if you like it. It's got tests that works for both your version and the new one/mine.

Reply:
Btw, moving it into a utility is a good call! 👍

Reply:
Awesome! I'll use this one as it's much cleaner :)